assets/xml/objects/object_fall2.xml

```diff
@@ -4,15 +4,15 @@
         moon's mouth, both used in the cutscene before warping to the moon.
     -->
     <File Name="object_fall2" Segment="6">
-        <DList Name="gOpenMouthMoonDL" Offset="0x2970" />
+        <DList Name="gOpenMouthMoonDL" Offset="0x2970" /> <!-- Original name is "moon_open" -->
         <Texture Name="gOpenMouthMoonFarSideTLUT" OutName="open_mouth_moon_far_side_tlut" Format="rgba16" Width="4" Height="4" Offset="0x34A8" />
         <Texture Name="gOpenMouthMoonFaceTLUT" OutName="open_mouth_moon_face_tlut" Format="rgba16" Width="4" Height="4" Offset="0x34C8" />
         <Texture Name="gOpenMouthMoonEyesTex" OutName="open_mouth_moon_eyes" Format="rgba16" Width="32" Height="32" Offset="0x34E8" />
         <Texture Name="gOpenMouthMoonFarSideTex" OutName="open_mouth_moon_far_side" Format="ci4" Width="64" Height="64" Offset="0x3CE8" />
         <Texture Name="gOpenMouthMoonFaceTex" OutName="open_mouth_moon_face" Format="ci4" Width="64" Height="64" Offset="0x44E8" />
         <Texture Name="gOpenMouthMoonTeethTex" OutName="open_mouth_moon_teeth" Format="rgba16" Width="64" Height="32" Offset="0x4CE8" />
         <!-- <Blob Name="object_fall2_Blob_005CE8" Size="0x20C" Offset="0x5CE8" /> -->
-        <Blob Name="object_fall2_Blob_005EF4" Size="0x1C" Offset="0x5EF4" />
+        <Blob Name="object_fall2_Blob_005EF4" Size="0x1C" Offset="0x5EF4" /> <!-- Original name might be "moon_start_anm"; either this or the one below is named that. -->
```

Comment:
We.. still dont know what it is then? is _anm a common pattern?

Reply:
Yeah, we don't know which one it is. There was a brief discussion on it here: https://discord.com/channels/688807550715560050/688851976746041391/1079347484842807307

Because this uses the `c_keyframe` stuff, we don't quite know which of these is the animation. *One* of them has to be, but we don't know which one. So I just did both. And no, I've actually never seen `_anm`; it might be a pattern used for keyframe animation, or it might just be the devs being inconsistent again.

Reply:
Could you add a comment saying this is related to the `c_keyframe` system for future reference?

Reply:
Good point, I added that to the comment.